### PR TITLE
updating docs with new release information

### PR DESCRIPTION
**Pull request by iporollo**
Docs updates for latest release

### CODE REVIEW

File: documentation/changelog/2022-04-13-v011-release.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+tags:
+  - releases
+posted_on_: 2022-04-13T00:00:00Z
+slug: latest-release
```
Comment (unkrich) [inline documentation/changelog/2022-04-13-v011-release.md:5]: should probably make this v011 so we dont have to change it like the other

File: documentation/docs/integrations/webhooks.md [hunk 2]
```diff
@@ -4,4 +4,39 @@ title: Webhooks Integration
 sidebar_label: Webhooks
 ---
 
-More information on how to integrate with webhooks is coming soon. If you’re interested in this option, please see or open a [GitHub Issue](https://github.com/monosidev/monosi/issues).
+Monosi integrates directly with Slack to be alert you in real-time of anomalies. It does so through the Incoming Webhooks feature of Slack.
```
Comment (unkrich) [inline documentation/docs/integrations/webhooks.md:7]: This page is not the same as the slack incoming webhook

**Review by unkrich — DISMISSED** [review]
Approving, but still requesting some changes

File: documentation/changelog/2022-04-13-v011-release.md (hunk 1 above)
Comment (iporollo) [reply documentation/changelog/2022-04-13-v011-release.md]: fixed

File: documentation/docs/integrations/webhooks.md (hunk 2 above)
Comment (iporollo) [reply documentation/docs/integrations/webhooks.md]: fixed

**Review by unkrich — APPROVED** [review]

**Review by unkrich — APPROVED** [review]

**Pull request merged**